plugin/mutate/source/dextool/plugin/mutate/backend/report/html/package.d

```diff
@@ -667,6 +671,48 @@ void generateFile(ref Database db, ref FileCtx ctx) @trusted {
     // newlines, detect when a line changes etc.
     auto lastLoc = SourceLoc(1, 1);
 
+    // read coverage data and save covered lines in lineList
+    auto id = ctx.fileId;
+    auto dbData = db.coverageApi.getCoverageStatus[id];
+    
+    auto byteCounter = 0;
+    auto lineCounter = 1;
+    auto r = [0];
+    auto lineList = assocArray(r, true.repeat);
+    alias T = Tuple!(int, "value", bool, "status");
```

Comment:
for simple tuple types it is prettier to create a struct. so you can replace this line with:

```d
static struct T { int value; bool status }
```